recipes/intel-ipsec-mb/all/conanfile.py

```diff
@@ -0,0 +1,82 @@
+from conans import tools, ConanFile, AutoToolsBuildEnvironment
+
+required_conan_version = ">=1.39.0"
+
+class IntelIpSecMbConan(ConanFile):
+    name = "intel-ipsec-mb"
+    description = "Intel(R) Multi-Buffer Crypto for IPSec"
+    license = "BSD 3-Clause"
+    homepage = "https://github.com/intel/intel-ipsec-mb"
+    url = "https://github.com/conan-io/conan-center-index"
+    topics = ("intel", "IPSec", "IPSec_MB")
+
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        print(self.conan_data["sources"][self.version])
+        tools.get(**self.conan_data["sources"][self.version],
+                  destination=self._source_subfolder, strip_root=True)
+
+    def _configure_autotools(self):
```

Comment:
Needed my suggestion in `build_requirements`.
```suggestion
                  destination=self._source_subfolder, strip_root=True)

    @property
    def _settings_build(self):
        getattr(self, "settings_build", self.settings)

    def _configure_autotools(self):
```